libraries/net/node.cpp

```diff
@@ -280,6 +282,73 @@ namespace graphene { namespace net { namespace detail {
 #define MAXIMUM_NUMBER_OF_BLOCKS_TO_HANDLE_AT_ONE_TIME 200
 #define MAXIMUM_NUMBER_OF_BLOCKS_TO_PREFETCH (10 * MAXIMUM_NUMBER_OF_BLOCKS_TO_HANDLE_AT_ONE_TIME)
 
+   class random_address_builder : public node_impl::address_builder
```

Comment:
This is pretty useless IMO. Clients can call this repeatedly to collect a list of all connected peers, eventually.
Randomizing on current time even allows the client to determine which entries of our list it has already seen and which are still missing.

Reply:
I agree. Removed